modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/endpoint/AbstractAssetManagerRestEndpoint.java

```diff
@@ -165,6 +166,43 @@ public Response snapshot(@FormParam("mediapackage") final MediaPackageImpl media
     }
   }
 
+  @POST
+  @Path("updateIndex")
+  @RestQuery(name = "updateIndex",
+      description = "Trigger search index update for event. The usage of this is limited to global administrators.",
+      restParameters = {
+          @RestParameter(
+              name = "id",
+              isRequired = true,
+              type = STRING,
+              description = "The event ID to trigger an index update for.")},
+      responses = {
+          @RestResponse(
+              description = "Update successfully triggered.",
+              responseCode = SC_NO_CONTENT),
+          @RestResponse(
+              description = "Not allowed to trigger update.",
+              responseCode = SC_FORBIDDEN),
+          @RestResponse(
+              description = "No such event found.",
+              responseCode = SC_NOT_FOUND),
+          @RestResponse(
+              description = "There has been an internal error and no snapshot could be taken.",
```

Comment:
No snapshot is being taken in any case :)

Reply:
While this is still technically correct :grin:… you are absolutely right that this doesn't make sense. I've updated the RESR docs.